### PR TITLE
Ll merge

### PR DESCRIPTION
**Pull request by Ginsusamurai**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code-challenges/401/linkedList/lib/ll.js [hunk 1]
```diff
@@ -191,6 +192,35 @@ class LinkedList {
...
+  while (list1Current !== null || list2Current !== null){
+    
+    list1Next = list1Current.next;
+    list2Next = list2Current.next;
+
+    list1Current.next = list2Current;
+    list2Current.next = list1Next;
+
+    list1Current = list1Next;
+    list2Current = list2Next;
```
Comment (adrianhuebner) [inline code-challenges/401/linkedList/lib/ll.js:218]: You can add some if conditionals here that say if this is null, then next is sent to the other list
   - I would do this for both lists
